src/JPNIC.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license isOsiApproved="false" licenseId="JPNIC" name="Japan Network Information Center License">
+    <crossRefs>
+      <crossRef>https://gitlab.isc.org/isc-projects/bind9/blob/master/COPYRIGHT#L366</crossRef>
+    </crossRefs>
+    <text>
+      <titleText>
+        <p>Japan Network Information Center License</p>
+      </titleText>
+      <copyrightText>
+        <p>Copyright (c) <alt name="copyright" match=".+">2000-2002 Japan Network Information Center</alt>. All rights reserved.</p>
+      </copyrightText>
+      <p>By using this file, you agree to the terms and conditions set forth bellow.</p>
```

Comment:
Hah, I hadn't noticed before but just happened to see here the typo in the last word for this sentence.  =)  Which is a typo from upstream, so it's fine that it's reproduced here.

If we're preparing this as a template that others might use, it might be worth changing the last word to the following, so that it'll still match in case someone notices the typo and fixes it in their own use of the license:

`<alt match="bellow|below" name="bellow">bellow</alt>`

cc @goneall and @zvr in case I'm getting the formatting wrong here...

Reply:
```suggestion
      <p>By using this file, you agree to the terms and conditions set forth <alt match="bellow|below" name="bellow">bellow</alt>.</p>
```